### PR TITLE
#49 fetch schema from core repo (latest tag)

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
add a vsix asset on release

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -49,7 +48,7 @@ jobs:
               )
           """ > /tmp/check.py
           python /tmp/check.py "${{ steps.extract_branch.outputs.branch }}"
-
+
```
Comment (FabienLelaquais) [inline .github/workflows/release.yml]: Empty lline shoudln't it be?

File: l10n/bundle.l10n.json [hunk 2]
```diff
@@ -26,6 +26,7 @@
   "Using TOML Schema Validation from {0}": "Using TOML Schema Validation from {0}",
   "Validation Schema not found in package. {0}": "Validation Schema not found in package. {0}",
   "Using embedded TOML Schema Validation": "Using embedded TOML Schema Validation",
+  "Using non fetched embedded TOML Schema Validation": "Using non fetched embedded TOML Schema Validation",
```
Comment (FabienLelaquais) [inline l10n/bundle.l10n.json:29]: What does this message mean?

**Review by FabienLelaquais — APPROVED** [review]
I'd be glad to discuss information message clarity though.
And I'm still 🤔 on how this works.

File: .github/workflows/release.yml (hunk 1 above)
Comment (FredLL-Avaiga) [reply .github/workflows/release.yml]: no big deal but ok

File: .github/workflows/prerelease-publish.yml [hunk 3]
```diff
@@ -56,6 +56,17 @@ jobs:
...
+      - name: get core latest version tag
+        run: echo "core_version=$(git ls-remote --tags --refs --sort='-version:refname' https://github.com/Avaiga/taipy-core [0-9]\* | head -1 | cut --delimiter='/' --field=3)" >> $GITHUB_OUTPUT
+        id: extract_core_version
+
+      - name: Download core schema file
+        working-directory: ./schemas
+        run: |
+            [ "${{ steps.extract_core_version.outputs.core_version }}" != "" ]
+            curl --fail-with-body https://raw.githubusercontent.com/Avaiga/taipy-core/${{ steps.extract_core_version.outputs.core_version }}/src/taipy/core/config/config.schema.json -o core.config.schema.json
+
```
Comment (jrobinAV) [inline .github/workflows/prerelease-publish.yml:69]: Looks good to me. I believe the file location and the branch are correct.

I did not test the curl, though. Do you want me to run it?

File: l10n/bundle.l10n.json (hunk 2 above)
Comment (FredLL-Avaiga) [reply l10n/bundle.l10n.json:29]: the schema is fetched from the core repo at package time as core.config.schema.json
when running in dev mode or building locally, the core file is not present and we have a failover file for tests 
this log is shown in that case => we can check which file is used in the log

**Review by jrobinAV — APPROVED** [review]
Looks good to me

File: .github/workflows/prerelease-publish.yml (hunk 3 above)
Comment (FredLL-Avaiga) [reply .github/workflows/prerelease-publish.yml:69]: I ran it several times in a now-deleted branch

File: l10n/bundle.l10n.json (hunk 2 above)
Comment (FabienLelaquais) [reply l10n/bundle.l10n.json:29]: I kind of get it. Thx

**Pull request merged**